app/assets/javascripts/studyfinder.js

```diff
@@ -83,7 +83,9 @@ function applyTypeahead() {
     },
     {
       name: 'keyword-search',
-      displayKey: 'text',
+      displayKey: function(suggestion) {
```

Comment:
where is this coming from or what is it doing? It appears to just be returning whatever is passed to it?

Reply:
Yep! It's because this old version of Twitter Typeahead doesn't handle just an array of strings. It's expecting objects with keys. If you want to to just use the values, you *have* to do this.

The current version of Typeahead handles this case by default and I have a branch that updates it, but it was more invasive than I wanted to deal with right now.